src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/CheckPowTransactionRuleTest.cs

```diff
@@ -49,7 +49,7 @@ public void CheckTransaction_TransactionOverSized_ThrowsBadTransactionOversizeCo
         {
             var transaction = new Transaction();
             transaction.Inputs.Add(new TxIn());
-            transaction = GenerateTransactionWithWeight(transaction, this.options.MaxBlockBaseSize + 1, NetworkOptions.TemporaryOptions & ~NetworkOptions.Witness);
+            transaction = this.GenerateTransactionWithWeight(transaction, this.options.MaxBlockBaseSize + 1, TransactionOptions.None);
```

Comment:
I'm not familiar with this code but is this change here correct?  Just flagging it as it noticably different to the original that does a bitwise compare of two of the options

Reply:
Yes the original would result in None, and the code before the change to add NetworkOptions was actually None.